pganonymize/version.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 
-__version__ = '0.11.0'
+__version__ = '0.11.1'
```

Comment:
Only a small annotation: can you please remove the version change? Due to other pull request the upcoming version will be 0.12.0, but normally I only change the version within the `development` branch, right before creating a new release.